src/ima.js

```diff
@@ -886,10 +883,10 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
       if (typeof this.config.vastLoadTimeout === 'number') {
         adsRequest.vastLoadTimeout = this.config.vastLoadTimeout;
       }
-      if (typeof this.config.omSdkAccessModes === 'object') {
-        Object.keys(this.config.omSdkAccessModes).forEach(accessMode => {
-          adsRequest.omidAccessModeRules[accessMode] = this.config.omSdkAccessModes[accessMode];
-        });
+      if (typeof this.config.omidAccessModes === 'object') {
+        for (const [vendor: OmidVendor, accessMode: AccessMode] of Object.entries(this.config.omidAccessModes)) {
+          adsRequest.omidAccessModeRules[this._sdk.OmidVerificationVendor[vendor]] = this._sdk.OmidAccessMode[accessMode];
+        }
```

Comment:
```suggestion
adsRequest.omidAccessModeRules = {...adsRequest.omidAccessModeRules, ...this.config.omidAccessModes}
```

Reply:
you will end with the kies instead of the values 
"this._sdk.OmidVerificationVendor[vendor] " return numerous values